lib/aerospike.js

```diff
@@ -582,6 +582,14 @@ exports.setupGlobalCommandQueue = function (policy) {
   EventLoop.setCommandQueuePolicy(policy)
 }
 
+/**
+  * The {@link module:aerospike/batch_type|batch_type} module contains a list of the
+  * batch record types.
+  *
+  * @summary {@link module:aerospike/batch_type|aerospike/batch_type} module
+  */
+exports.batch_type = require('./batch_type')
```

Comment:
```suggestion
exports.batchType = require('./batch_type')
```

You could also just do `exports.batchType = as.batchTypes` here, similar to how all of the other enums are defined under the "Enumerations" section, e.g. `exports.indexType = as.indexType`: https://github.com/aerospike/aerospike-client-nodejs/blob/ca82e651c87862d6438c9f2b28050ef3c84b4ff9/lib/aerospike.js#L433-L453